projects/core/src/lib/classes/content-editable.ts

```diff
@@ -0,0 +1,71 @@
+import {MaskitoElement} from '../types/maskito-element';
+import {getContentEditableSelection} from '../utils/dom/get-content-editable-selection';
+import {setContentEditableSelection} from '../utils/dom/set-content-editable-selection';
+
+export class MaskitoContentEditable implements MaskitoElement {
+    // declared fields
+    declare addEventListener: HTMLElement['addEventListener'];
+    declare removeEventListener: HTMLElement['removeEventListener'];
+
+    declare dispatchEvent: HTMLElement['dispatchEvent'];
+    declare isContentEditable: HTMLElement['isContentEditable'];
+    declare nodeName: HTMLElement['nodeName'];
+    declare matches: HTMLElement['matches'];
+    declare ownerDocument: HTMLElement['ownerDocument'];
```

Comment:
Could we improve solution somehow and don't iterate all possible properties ?
Actually, plugins can use ANY possible property of `HTMLElement`

Reply:
Unfortunately, without fixing this problem we can't merge this solution.
Our users could write their own plugins and use ANY possible properties of `HTMLElement`.
It will cause breaking change for them(